pkg/tools/cli/loadtest/usecase/load.go

```diff
@@ -18,27 +18,52 @@ package usecase
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
+	"github.com/vdaas/vald/internal/client/gateway/vald/grpc"
 	"github.com/vdaas/vald/internal/errgroup"
 	"github.com/vdaas/vald/internal/runner"
-	"github.com/vdaas/vald/pkg/agent/ngt/config"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/config"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service/insert"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service/search"
 )
 
 type run struct {
 	eg  errgroup.Group
 	cfg *config.Data
+	l   service.Load
 }
 
 func New(cfg *config.Data) (r runner.Runner, err error) {
-	return nil, nil
+	run := new(run)
+	run.cfg = cfg
+	run.eg = errgroup.Get()
+
+	ctx := context.Background()
```

Comment:
It may be better to pass context from the upper layer.:thinking:

e.g.)
```go
func New(ctx context.Context.....)
```

Reply:
I migrate grpc client init code from New() to PreStart()

Reply:
sure